dev-java/annogen/annogen-0.1.0-r4.ebuild

```diff
@@ -12,19 +12,15 @@ SRC_URI="http://dist.codehaus.org/${PN}/distributions/${P}.zip"
 
 LICENSE="BSD"
 SLOT="0"
-KEYWORDS="amd64 ~ppc64 ~x86"
+KEYWORDS="~amd64 ~ppc64 ~x86"
```

Comment:
here you're dropping the package to unstable and later you remove the stable version. keep `amd64` stable in this ebuild and if my tests pass, i will commit it straight to stable.